graylog2-web-interface/src/components/field-analyzers/QuickValuesOptionsForm.jsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import _ from 'lodash';
+import { Button, Col, Row } from 'react-bootstrap';
+import { Input } from 'components/bootstrap';
+import FormsUtils from 'util/FormsUtils';
+
+const QuickValuesOptionsForm = React.createClass({
+  propTypes: {
+    limit: PropTypes.number.isRequired,
+    tableSize: PropTypes.number.isRequired,
+    order: PropTypes.string.isRequired,
+    onSave: PropTypes.func.isRequired,
+  },
+
+  getInitialState() {
+    return {
+      limit: this.props.limit,
+      tableSize: this.props.tableSize,
+      order: this.props.order,
+    };
+  },
+
+  _onChange(event) {
+    const state = _.cloneDeep(this.state);
+    state[event.target.name] = FormsUtils.getValueFromInput(event.target);
+    this.setState(state);
+  },
+
+  _onOrderChange(value) {
+    return (e) => {
+      e.preventDefault();
+      this._onChange(FormsUtils.inputEvent('order', value));
+    };
+  },
+
+  _onSave(e) {
+    e.preventDefault();
+    this.props.onSave(this.state);
+  },
+
+  render() {
+    return (
+      <Row>
+        <form className="form form-horizontal" onSubmit={this._onSave}>
```

Comment:
The `form` and `fieldset` tags should be inside `Col`, as is intended by bootstrap. Otherwise we may ran in some CSS issues.

Reply:
There's no cancel button on this form and no obvious way of getting back to the previous screen. I would add a `Cancel` button at the end, to make it more obvious.

Reply:
I would also like to get rid of the `form-horizontal` here and make the column size smaller. In my opinion the single column form avoids making your eyes jump from the label to the field and in this case we don't have that many inputs anyway:

<img width="1261" alt="screen shot 2017-10-04 at 15 28 06" src="https://user-images.githubusercontent.com/716185/31178232-abe1b1e8-a918-11e7-91a6-11abd8487ccf.png">

At the very least we should reduce the column size, because the current inputs don't require that much space, and I think long inputs look even worse than some white empty space.

Reply:
In the example I did above, I also used a bit of padding (15px), so it looks aligned with the pie chart in the quick values visualization.